katello.te

```diff
@@ -51,6 +54,9 @@ require{
 # Allow FFI library callbacks - see RM#26520 for more info
 allow passenger_t self:process execmem;
 
+# Connections to Candlepin
+allow passenger_t katello_candlepin_port_t:tcp_socket name_connect;
```

Comment:
I guess this needs to be puma these days? Or both?

Reply:
See the other PR, then I will rebase this PR on top of that one so yes - this will become a puma rule.